src/transactional.ts

```diff
@@ -1,6 +1,6 @@
 import { Service } from "./service";
 
-export function Transactional() {
+export function Transactional(dataSource?: string) {
```

Comment:
나중에 무슨 옵션이 추가로 들어갈지 몰라도 추가될수 있으니까 object로 받는게 나을듯 합니다

```suggestion
export function Transactional(options?: { dataSource?: string }) {
```